packages/cozy-client/src/models/dacc.js

```diff
@@ -71,3 +80,45 @@ export const sendMeasureToDACC = async (client, remoteDoctype, measure) => {
     throw error
   }
 }
+
+export const buildAggregateParams = params => {
+  const { measureName } = params
+  if (!measureName || typeof measureName !== 'string') {
+    throw new Error('Missing or wrong type parameter: measureName')
+  }
+  const startDate = params.startDate || new Date(0).toISOString()
+  const endDate = params.endDate || new Date(Date.now()).toISOString()
+  if (!isCorrectDateFormat(startDate) || !isCorrectDateFormat(endDate)) {
+    throw new Error('Date should be in YYYY-MM-DD format')
+  }
+  return { measureName, startDate, endDate }
+}
+
+/**
+ * Send measures to a DACC through a remote doctype
+ *
+ * @param {object} client - The CozyClient instance
```

Comment:
You can do something like: 
```
import CozyClient from '../CozyClient'

@param {CozyClient} client
```

I don't know why CozyClient is not in the types yet.